packages/deployer/tasks/deploy.js

```diff
@@ -50,4 +58,20 @@ task(TASK_DEPLOY, 'Deploys all system modules')
     await hre.run(SUBTASK_DEPLOY_ROUTER);
     await hre.run(SUBTASK_UPGRADE_PROXY);
     await hre.run(SUBTASK_FINALIZE_DEPLOYMENT);
+
+    _forceSilenceHardhat(false);
   });
+
+/*
+ * Note: Even though hardhat's compile task has a quiet option,
+ * it stil prints some output. This is a hack to completely silence
+ * output during deployment.
+ * */
+function _forceSilenceHardhat(silence) {
+  if (silence) {
+    logCache = console.log;
+    console.log = () => {};
+  } else {
+    console.log = logCache;
```

Comment:
If quiet is false it will end with console.log = undefined

Reply:
Fixed